### PR TITLE
Start using CircleCi

### PR DESCRIPTION
**Pull request by TomasKorbar**
Hi,
as we discussed Travis is not suitable for multiple checks which we need. CircleCI will provide us easily extensible platform for future testing.
Feel free to propose any change in configuration file.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            pip install pylint
+      - run:
+          name: download tool for comparing of pylints outputs
+          command: |
+            git clone https://github.com/TomasKorbar/pylint_regexer ./regexer
+      - run:
+          name: check current code with pylint
+          command: |
+            . venv/bin/activate
+            pylint -E --output-format=parseable -s n --exit-zero ./tuned/ > /tmp/new_pl_output
```
Comment (olysonek) [inline .circleci/config.yml]: This checks only the `tuned` module, right? Ideally, it should check also the python scripts in the top level directory and tests. Take a look at the `lint` target in the Makefile.

**Comment by TomasKorbar** [comment]
Commits squashed. We are good to go.

**Comment by yarda** [comment]
Thanks, but we finally ended on Packit (well, it currently doesn't work, but this is different story :), so I think this should be stripped down just to Travis removal.

**Comment by olysonek** [comment]
Hm, I completely forgot that we currently have a travis config in `master`. It doesn't seem to run for pull requests, though..

Remember that we don't currently run unit tests in Packit, and I'm not sure Packit is a good solution for that. Perhaps we might like to keep Travis..

**Comment by yarda** [comment]
> Hm, I completely forgot that we currently have a travis config in `master`. It doesn't seem to run for pull requests, though..
> 
> Remember that we don't currently run unit tests in Packit, and I'm not sure Packit is a good solution for that. Perhaps we might like to keep Travis..

We run unit tests in Packit, they are run during the build in copr.

**Comment by olysonek** [comment]
> We run unit tests in Packit, they are run during the build in copr.

I forgot about that. I'm ok with removing travis.

**Comment by TomasKorbar** [comment]
I suppose we do not need the dockerfile in tests folder anymore too, am i right?

**Comment by olysonek** [comment]
> I suppose we do not need the dockerfile in tests folder anymore too, am i right?

I guess not.

**Comment by TomasKorbar** [comment]
Then this is ready to be merged.

**Comment by yarda** [comment]
Thanks, merging.

**Pull request merged**